View.md

```diff
@@ -178,6 +178,24 @@ Backward: Use `%{input}` and `%{line}` as placeholders.
 ],
 ```
 
+##### [Okular](https://apps.kde.org/en/okular) support 
+
```

Comment:
I think you should cite the source  https://miterion.de/post/vscodeplusokular/

Reply:
sure